pkg/epinio/components/application/AppSource.vue

```diff
@@ -396,8 +397,15 @@ export default Vue.extend<Data, any, any, any>({
       }));
     },
 
+    sourceValue() {
+      return {
+        ...this.source.git,
+        type: toLabel(this.type),
+      };
+    },
+
     type() {
-      // There's a bug in the select component which fires off the option ({ value, label}) instead of the value
+      // TODO There's a bug in the select component which fires off the option ({ value, label}) instead of the value
```

Comment:
TODO

Reply:
Removed `unSafeType` variable. Do you remember in what case the selected option is of type` ({ value, label})`. I did several tests but I am not able to replicate the bug in this PR.

Reply:
Could have been fixed in upstream. If you've tested each source then we can keep your changes